float64[10] angle2motor
